validator/tests/test_scheduler/tests.py

```diff
@@ -162,10 +213,158 @@ def test_serial_batch_fails_valid_txn(self):
             context_manager=context_manager,
             name='batch_fails_valid_txn.yaml')
 
+    def test_parallel_complex_batches_multiple_failures(self):
+        """Tests the parallel scheduler against the
+        test_scheduler/data/complex_batches_multiple_failures.yaml file.
+
+        Notes:
+            This yaml file has txns that depend on txns in several batches
+            and several failed batches.
```

Comment:
Please add some more description of what this test is doing (along with the subsequent tests)